frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -164,14 +163,6 @@ export default class PivotTable extends Component {
   };
 
   render() {
-    // We need to tell the List/Grids to call the columnWidth/rowHeight functions again when data changes.
-    // Putting this in componentDidUpdate led the dimensions to be recomputed _after_ re-rendering the cells.
-    // According to the docs, recomputing dimensions should force a render but this didn't occur correctly.
-    // The downside of keeping it here is that the dimensions are computed twice per render.
-    this.bodyGrid && this.bodyGrid.recomputeGridSize();
-    this.topGrid && this.topGrid.recomputeGridSize();
-    this.leftList && this.leftList.recomputeRowHeights();
```

Comment:
:+1:  for not needing `recomputeGridSize` anymore.

Reply:
yes! A nice unexpected advantage of the smaller virtual cells. The Container/Grid props will always change so we don't need to force a recomputation.